R/contribution_matrix.R

```diff
@@ -813,6 +817,9 @@ CreateContributionMatrix <- function(data, treatment_ids, outcome_type, outcome_
   treatments <- treatment_ids$Label
   #Unduplicated covariate values (one per study)
   covariate <- unique(dplyr::select(data, starts_with(c("Study", "covar."))))$covar.
+  if (any(is.na(covariate))) {
+    stop("Missing covariate values are not allowed")
```

Comment:
There were missing values early on in the task before I updated GetReferenceOutcome( ) to get the imputed values. So I think you're right that this isn't necessary anymore. I'd like to leave it in case the function is used outside MetaInsight.
I believe it just showed the red message when there were missing values. Actually I had to remove the tryCatch statement when doing the task because I couldn't see the error message when anything went wrong.